src/Chapter2.hs

```diff
@@ -336,7 +336,7 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList b e l = if (b < 0 || e < 0) then [] else take (e-b+1) (drop b l)
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check :slightly_smiling_face: